electron/gog/setup.ts

```diff
@@ -137,17 +182,33 @@ async function setup(appName: string): Promise<void> {
             gameInfo.install.install_path
           )
           const type = actionArguments.type
+          const sourcePath = actionArguments?.source
```

Comment:
same here with the `source` since it depends on it to get going.

Reply:
I tested both cases you mentioned. sourcePath will just be undefined. And since replace is not a memeber of undefined it won't get executed.

Reply:
will not crash with message replace is not a function of undefined ?